dist/config/doctrine.yaml

```diff
@@ -0,0 +1,18 @@
+parameters:
+    env(MYSQL_HOST): 'database'
+    env(MYSQL_PORT): '3306'
+    env(MYSQL_USER): 'pimcore'
+    env(MYSQL_PASSWORD): 'pimcore'
+    env(MYSQL_DATABASE): 'pimcore'
+
+doctrine:
+    dbal:
+        server_version: '%env(MYSQL_SERVER_VERSION)%'
```

Comment:
Nice, but then don't we need to remove that as well? I'd expect Symfony to fail if the `MYSQL_SERVER_VERSION` env variable does not exist. Haven't tried it, though. Have you?

Reply:
worked for me without setting the variable